Author: Diane Trout <diane@ghic.org>
Description: The find_package_handle_standard_args test
 wasn't passing for me with the swig 2.0.0 and cmake 2.8.11
 According to the cmake documentation, the purpose of 
 f_p_h_s is to set the <name>_FOUND variable if all the 
 listed variables are set. 
 So it seems silly to test for the variable that's going to 
 be set.


--- a/cmake/modules/FindSWIG.cmake
+++ b/cmake/modules/FindSWIG.cmake
@@ -1,9 +1,7 @@
 find_program(SWIG swig /usr/bin/)
 if(SWIG)
-    set(SWIG_FOUND ON)
     message("SWIG found")
 endif()
 
 #abort if any of the requireds are missing
-find_package_handle_standard_args(SWIG  DEFAULT_MSG
-                                  SWIG_FOUND SWIG)
\ No newline at end of file
+find_package_handle_standard_args(SWIG  DEFAULT_MSG SWIG)
